src/frame.js

```diff
@@ -0,0 +1,29 @@
+function Frame (seqNum) {
```

Comment:
I've yet to use function and prototype in this way (i used class, which i understand to be a newer addition to JS), so good job on understanding the protoype stuff!

Reply:
My plan was to set this up as a function and then add perhaps incorporate it into a separate bowlingGame class or something like that - hopefully it would of worked that way.